src/components/sequencing/SequenceEditor.svelte

```diff
@@ -6,6 +6,7 @@
   import { userSequencesRows } from '../../stores/sequencing';
   import type { Monaco, TypeScriptFile } from '../../types/monaco';
   import effects from '../../utilities/effects';
+  import CommandDictJson from '../../workers/command_dict.json?raw';
```

Comment:
It looks like there's already some logic for fetching some command dictionary files using the `sequenceCommandDictionaryId`. Were those not sufficient for this?

In general, I don't think we should be using hardcoded command dictionaries in the code since this is meant to be generic enough to handle any model/dictionary. This new `command_dict.json` file also contains "Banananation" specific entries, which I don't think should referenced directly because it is a dummy model.

Reply:
Yep, 100%!! This was a demo because I didn't have time to get the data piped all the way - through from the new backend changes, so this is just here until that gets glued in. I'm running out of time in my internship so someone else might need to help get that part over the line. Still, I'll try to take a crack later this week, schoolwork permitting 🤞

Reply:
This is because we didn't have the endpoint hooked up yet so @Cobular couldn't fetch that data. I am going to make a ticket and hook up the endpoint for the UI to fetch.

Reply:
The data is available in the database already.